<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFormDoc</class>
 <widget class="QWidget" name="TFormDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="statusTip">
    <string extracomment="打开文件">打开文件</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="statusTip">
    <string extracomment="剪切">剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="statusTip">
    <string extracomment="复制">复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="statusTip">
    <string extracomment="粘贴">粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>设置字体</string>
   </property>
   <property name="statusTip">
    <string extracomment="字体">字体</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
   <property name="statusTip">
    <string extracomment="关闭">关闭</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销编缉操作</string>
   </property>
   <property name="statusTip">
    <string extracomment="撤销">撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>重复</string>
   </property>
   <property name="toolTip">
    <string>重复编缉操作</string>
   </property>
   <property name="statusTip">
    <string comment="重做" extracomment="重做">重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>TFormDoc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
